content/docs/clients/javascript/node-example.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Node.js Example
+# order: 7
+# parent: JS
+---
+When working with Node.js in a non-server environment means you have to think about queue processing a little more than in any other environment. If your Node.js app crashes or exits before the Exceptionless event queue can be cleared, you'll miss events. So, here is a simple guide to ensure you capture all events. 
+
+```js
+import { Exceptionless } from "@exceptionless/node";
+
+(async () => {
```

Comment:
I guess you are doing this because of the awaits and some platform not supporting top-level awaits. I think most platforms do support top level awaits and I think it would be better to omit the self-executing function to keep things simpler here. Also, I am pretty sure you could just not use the await at all and just do `Exceptionless.startup("YOUR API KEY")` and it would be fine. JavaScript promises aren't like C# ones where if you don't await them they won't even be started.

On that note @niemyjski, I don't love that we are using await's all over the place. I know we did it for the plugins and allowing those to be async, but it doesn't feel as simple as I'd like it to be.

Reply:
Node.js introduced top-level await in 14.3 but you have to modify the package.json because it hasn't received full support. I tend not to document that path for people in my own stuff, but if you want to just show top-level await and let people figure it out, I'm down. 

Also, if await isn't necessary, let's definitely drop it. Just let me know.